system_baseline/views/v1.py

```diff
@@ -222,14 +141,12 @@ def get_baselines(limit, offset, order_by, order_how, display_name=None):
     query = query.limit(limit).offset(offset)
     query_results = query.all()
 
-    return _build_paginated_baseline_list_response(
-        len(query_results),
-        limit,
-        offset,
-        order_by,
-        order_how,
-        query_results,
-        withhold_facts=True,
+    json_list = [baseline.to_json(withhold_facts=True) for baseline in query_results]
+
+    total_available = _get_total_baseline_count()
```

Comment:
The total baseline count should be the count of everything returned by the query if limit + offset wasn't there. I would remove this method and instead, capture it via `total_available = query.count()` above